src/hb-ot-layout.cc

```diff
@@ -966,6 +1348,15 @@ hb_ot_layout_lookup_would_substitute (hb_face_t            *face,
   return l.would_apply (&c, &face->table.GSUB->accels[lookup_index]);
 }
 
+
+/**
+ * hb_ot_layout_substitute_start:
+ * @font: #hb_font_t to use
+ * @buffer: #hb_buffer_t buffer to work upon
+ *
+ * Begins  
```

Comment:
Begins what?

Reply:
Good question :) -- I had meant to leave this one blank (as with a couple of other descriptors I need to tackle separately). Cleared it out for now.